### PR TITLE
Remove Ports from Role CRD

### PR DESCRIPTION
**Pull request by fhennig**
## Description

Without the agent, ports don't need to be configurable anymore

## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)

### CODE REVIEW

File: rust/operator-binary/src/config.rs [hunk 1]
```diff
@@ -175,7 +183,7 @@ pub fn get_runtime_properties(
     };
     let others =
         stackable_operator::product_config::writer::to_java_properties_string(other_props.iter());
-    format!("{}\n{}\n{}", common, role_specifics, others.unwrap())
+    format!("{}{}{}{}", ports, common, role_specifics, others.unwrap())
```
Comment (maltesander) [inline rust/operator-binary/src/config.rs:186]: Go for Result and "?" here instead of unwrap?

**Review by maltesander — CHANGES_REQUESTED** [review]
Just one unwrap comment and the zookeeper references in the custom resource examples go for "simple" zk cluster. In the zookeeper example its called "simple-zk".

**Review by maltesander — APPROVED** [review]
LGTM.

**Comment by fhennig** [comment]
bors merge

**Pull request merged**